docs/pbc/all/payment-service-providers/unzer/howto-tips-use-cases/understand-payment-method-in-checkout-process.md

```diff
@@ -9,11 +9,11 @@ Depending on your Unzer credentials type defined in the Back Office, the display
 
 ## Standard type credentials
 
-Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods that are currently enabled in your account. This list is compared against the active payment methods for this store. All payment methods that are at the intersection of the Unzer data and the Spryker data are displayed.
+Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. This list is compared against the active payment methods for this store. All payment methods at the intersection of the Unzer data and the Spryker data are displayed.
 
 ## Marketplace type credentials
 
-Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods that are currently enabled in your account. This list is compared against the active payment methods for this store and all the payment methods enabled for all the merchants inside the cart (defined in Unzer).
+Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. We compare the list against the active payment methods of the store and all the payment methods enabled for the merchants (in Unzer) inside the cart.
```

Comment:
```suggestion
Whenever a customer goes through the checkout process, Unzer is called to know all the payment methods currently enabled in your account. We compare the list against the active payment methods of the store and all the payment methods enabled for the merchants (in Unzer) related to the added products inside the cart.
```

Reply:
This is implied because the products belong to the merchants